packages/trader/src/Modules/Trading/Components/Form/screen-small.jsx

```diff
@@ -63,25 +69,32 @@ const CollapsibleTradeParams = ({
                     <BarrierMobile />
                 </div>
             )}
+            {isVisible('barrier_selector') && (
+                <div collapsible='true'>
```

Comment:
@NikitK-deriv it's an old code which is currently being improved by our quality team, collapsible accepts string value only because it was designed as a data-attribute long time ago. we won't touch it in our PR.